lerna.json

```diff
@@ -1,6 +1,7 @@
 {
   "packages": [
-    "packages/*"
+    "packages/*",
+    "examples/cdk"
```

Comment:
Since we are importing the published versions why is this needed?

Reply:
The idea is to take advantage of the same quality checker we have in modules: husky, lint and test with lerna locally and in GH actions etc.

Reply:
oh good point, thanks for explaining!